### DISCUSSION TITLE
brauche Tipp für Wifi Access Point Verbindungsprobleme / need tip for Wifi Access Point connection problems

### DISCUSSION THREAD

**Discussion by entfernt** [tbnobody/OpenDTU#408] [Q&A]
Hallo,

ich bin ESP Anfänger und kämpfe grade mit der Access Point Verbindung für die Erstkonfiguration von OpenDTU
Das WiFi Netz taucht auf  "OpenDTU-392...." aber die einzige Rückmeldung die ich beim Verbindungsversuch bekommen ist: Verbindung nicht möglich. (PCs und Smartphone)

Gerät ist ein ESP-32S TYPEC

Hat jemand eine Idee für mich?

Danke :) 

Hello,

I'm an ESP beginner and I'm struggling with the Access Point connection for the initial configuration of OpenDTU.
The WiFi network shows up "OpenDTU-392...." but the only feedback I get when trying to connect is: connection not possible. (PCs and smartphone)

Device is an ESP-32S TYPEC

Does anyone have any idea for me?

Thanks :) 

Build ist from here  #378

**Comment by Solarteur** [comment]
Bis zu welchem Schritt in "README" im Kapitel "First configuration" bist Du denn gekommen, bevor die Fehlermeldung erschien?

**Reply by entfernt** [reply]
> Bis zu welchem Schritt in "README" im Kapitel "First configuration" bist Du denn gekommen, bevor die Fehlermeldung erschien?


Danke für die Antwort.
->
naja ich stecke beim ersten ^^  wifi ssid tauch auf den Endgeräten auf .. Passwort kann ich eingeben.. drücke auf verbinden. -> "verbindung nicht möglich"

**Comment by entfernt** [comment]
Ich kann mir halt nicht vorstellen das mein ESP Hardwareseitig kaputt ist so dass ich es flashen kann und ein Netzwerk auftaucht ich aber nicht verbinden kann. das deutet doch eher auf ein Softwareproblem hin...  oder halt das "falsche" ESP Board ..  ist ein Aitexm ESP-32S TYPEC 38P 
ich hab auch schon den wifi scanner draufgeflashed der läuft und findet Netzwerke.
Evtl gehts wenn ich mein Wifi hardcode so dass ich den AP Schritt umgehe..  allerdings muss ich da erstmal den code durchsehen...

**Comment by Ascendor** [comment]
Was mir (spät) aufgefallen war: Ich hatte ein sehr langes WLAN-Passwort. Die Eingabemaske ist/war auf für meinen Fall "zu wenige" Zeichen limitiert. Ich hatte mein WLAN-Passwort dann geändert, und schon ging's. 
Ich weiß nicht ob das bei der aktuellen Firmware auch noch eine Ursache sein könnte.

**Reply by Solarteur** [reply]
Hier geht es um das Passwort für den AP - und das ist "openDTU42"....

**Reply by Ascendor** [reply]
Oh, entschuldigt, da habe ich nicht aufmerksam gelesen.

**Reply by entfernt** [reply]
> Hier geht es um das Passwort für den AP - und das ist "openDTU42"....

nur um missverständnissen Vorzubeugen :)  das Passwort ist denke ich nicht das Problem.. hab sogar mal ein einfacheres reinkompiliert auch ohne erfolg..

**Reply by tbnobody** [reply]
Hinweis:
1. Das Passwort muss mindestens 8 Zeichen haben sonst erfüllt es die WLAN Specification nicht.
2. Wenn du das Default Passwort im Source änderst musst du den ESP zuerst komplett löschen. Das Passwort wird beim ersten booten in die Config Datei geschrieben. Die Config Datei wird beim erneuten Flashen NICHT überschrieben.
3. Was sagt denn die Serielle Konsole wenn du versuchst dich per WiFi zu verbinden?

**Comment by fbnhffmnn** [comment]
Hallo zusammen,
eine kurze Frage, das flashen war erfolgreich. (via VisualStudio/PlatformIO Upload)  Leider kann ich den Access Point nicht finden, ich hatte das so verstanden, dass der dann anschließend auftaucht, wenn ich nach WLAN-Netzen suche. Aber leider ohne Ergebnis.

Habt ihr eine Idee, wie ich das Debuggen kann bzw. herausfinde, warum er den AccessPoint nicht aufspannt?

Vielen Dank und Grüße

**Reply by fbnhffmnn** [reply]
Ich hab es geschafft nach diversen permission Problemen, den Platformio Monitor zu starten. Ich habe leider keine Ahnung vom ESP32, das sollte mein erstes Projekt damit werden.

Vielleicht kann mit den Logs aber jemand etwas anfangen:

>21:59:47.847 > Starting OpenDTU
>21:59:47.848 > Initialize FS... done
>21:59:47.849 > Reading configuration... [    23][E][vfs_api.cpp:105] open(): /littlefs/config.json does not exist, no permits for creation
>21:59:47.860 > Failed to read file, using default configuration
>21:59:47.865 > done
>21:59:47.866 > Reading PinMapping... [    38][E][vfs_api.cpp:105] open(): /littlefs/pin_mapping.json does not exist, no permits for creation
>21:59:47.876 > using default config done
>21:59:47.879 > Initialize Network...
>21:59:47.914 > Brownout detector was triggered
>21:59:47.917 >
>21:59:47.918 > ets Jul 29 2019 12:21:46
>21:59:47.920 >
>21:59:47.920 > rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
>21:59:47.926 > configsip: 0, SPIWP:0xee
>21:59:47.927 > clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
>21:59:47.934 > mode:DIO, clock div:2
>21:59:47.936 > load:0x3fff0030,len:1184
v21:59:47.938 > load:0x40078000,len:13104
v21:59:47.940 > load:0x40080400,len:3036
v21:59:47.943 > entry 0x400805e4
v21:59:48.521 > E (603) esp_core_dump_flash: No core dum��ѥѥ���found!
v21:59:48.527 > E (603) esp_core_dump_flash: No core dump partition found!

Diese Zeilen kommen immer und immer wieder, es scheint in einer Schleife zu hängen. Weil irgendwie die Config und das PIN_Mapping nicht gefunden wird ... Nur wo muss ich was dagegen tun?

**Comment by tbnobody** [comment]
> Brownout detector was triggered

Spannungsversorgung zu schlecht

**Reply by fbnhffmnn** [reply]
Vielen Dank. Das ist ja ernüchternd... Ich habe jetzt diverse Kabel und Ladegeräte ausprobiert ... immer dasselbe. Ich hab auch schon den nRF24L01+ abgeklemmt, ohne Änderung.
Ist es möglich, dass der ESP irgendwo einen Kurzen hat? Der ihn in die Knie zwingt?
Werde mir wohl noch ein Board und ein RF Modul kaufen und testen ... Mit dem komme ich so nicht weiter. Falls ihr dazu noch weitere Erfahrungen habt, sagt gerne bescheid. 
Vielen Dank erstmal

**Reply by fbnhffmnn** [reply]
FYI
kurzes Feedback, ich habe denselben ESP von Espressif nochmal bestellt. Diesmal hat alles reibungslos geklappt. Das erste Board scheint irgendwelche kalten Lötstellen oder sowas zu haben. Das es beim aufbauen des Netzwerks zusammen bricht. 
Vielen Dank nochmal für die Analyse.
